src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -301,7 +294,7 @@ class DataFramePreviewRowItem {
                         sw.WriteLine(line);
 
                         if (reportProgress) {
-                            var newProgressValue = 90 * (double)read / (double)sr.BaseStream.Length;
+                            var newProgressValue = 90 * (double)read / sr.BaseStream.Length;
```

Comment:
AFAIK in C# unary operators have higher precedence than binary: https://msdn.microsoft.com/en-us/library/6a71f45d.aspx